setup.py

```diff
@@ -72,6 +73,7 @@
             'coverage',
             'coveralls',
             'flake8',
+            'sentry-sdk[flask]'
```

Comment:
I don't think this needs to be here.  Requirements should only show up one time in the `setup.py` file.  The `test` section should only include the packages needed to run the tests/coverage/linting.

Reply:
thanks - removed it :)